### PR TITLE
PLT-1346: ability to publish uber jars

### PR DESCRIPTION
**Pull request by ab666**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#Tue Sep 10 15:50:33 PDT 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-bin.zip
```
Comment (javidang) [inline gradle/wrapper/gradle-wrapper.properties:3]: will it be fine to use 6.7 here ? I had troubles setting up new projects 6+ versions.

**Review by javidang — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -32,5 +29,5 @@ allprojects {
     }
 
     group = 'io.token.sdk'
-    version = '2.12.24'
+    version = '2.12.25-SNAPSHOT'
```
Comment (drobinson123) [inline build.gradle:32]: -SNAPSHOT?

Comment (ab666) [reply build.gradle:32]: oops

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (ab666) [reply gradle/wrapper/gradle-wrapper.properties:3]: should be fine

**Pull request merged**